client/python/test/test_client.py

```diff
@@ -308,6 +308,18 @@ def test_upload_file_upload_route_does_not_exist(self):
                 client.submit(1, "foo", f.name, fn_index=0).result()
                 serialize.assert_called_once_with(1, "foo", f.name)
 
+    def test_state_without_serialize(self, stateful_chatbot):
```

Comment:
We can add a `serialize` argument to `connect` to use it in this test.

Reply:
Agreed, that would be better!

Reply:
Fixed